### PR TITLE
Elixir training path: Ecto [1-5]

### LINKED ISSUE #22: Elixir training path: Ecto [1-5]
### Content:

- Basics,
- Changesets,
- Executables
- Associations
- Querying

### PR DESCRIPTION
**Pull request by Euphorichuman**
Basics, changesets, executables, associations, querying

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1 @@
+/.vscode/
```
Comment (luishurtado) [inline .gitignore]: No newline at the end of the file. You can fix this easily in the code editor settings, I guess.

**Review by luishurtado — CHANGES_REQUESTED** [review]

File: .gitignore (hunk 1 above)
Comment (Euphorichuman) [reply .gitignore]: Yes, I have CRLF set in vscode, I don't understand why '\ n' is not set at the end. Although I still don't understand why it is considered a good style to put a new line as the end?

Comment (luishurtado) [reply .gitignore]: @Euphorichuman Google it and post your findings here. GitHub always shows a warning about that missing character.

Comment (Euphorichuman) [reply .gitignore]: @luishurtado I found this quite interesting: "A text file, under UNIX, consists of a series of lines, each of which ends with a newline character (`\n`). A file that is not empty and does not end with a newline is therefore not a text file."

For my case (Windows): On DOS and Windows, each CR LF sequence (`\ r \ n`) is mapped to a newline (`\ n`; this is what always happens when reading a file opened as text on these OSes). 

Source: https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline

Comment (luishurtado) [reply .gitignore]: OK. Now we have the reason. I highly recommend setting up a Linux machine for work. Sharing files between a Windows file system and a Linux OS in a container or similar always leads to weird problems.

**Review by luishurtado — APPROVED** [review]

**Pull request merged**